app/src/main/java/org/p2p/wallet/jupiter/repository/tokens/db/SwapTokensDao.kt

```diff
@@ -0,0 +1,49 @@
+package org.p2p.wallet.jupiter.repository.tokens.db
+
+import androidx.room.Dao
+import androidx.room.Insert
+import androidx.room.Query
+import org.intellij.lang.annotations.Language
+
+@Dao
+interface SwapTokensDao {
+    private companion object {
+        @Language("RoomSql")
+        private const val INDEX_FOR_ADDRESS =
+            "SELECT ordinal_index FROM swap_tokens WHERE address = :mintAddress"
+
+        @Language("RoomSql")
+        private const val SWAPPABLE_TOKEN_INDEXES_FOR_TOKEN =
```

Comment:
this can be rewritten without subquery:
```sql
SELECT swap_tokens_routes.index_swappable_token
FROM swap_tokens_routes
JOIN swap_tokens ON swap_tokens_routes.index_source_token = swap_tokens.ordinal_index
WHERE swap_tokens.address = :mintAddress
```

but it needs to be tested, I'm not sure which exactly `JOIN` is suitable here